### PR TITLE
Add some missing TABLES exports: keyProperties, clefProperties, durationToTicks

### PR DESCRIPTION
**Pull request by AaronDavidNewman**
Unless I am missing something these are not accessible anymore to programs using Vex as a library.

### CODE REVIEW

**Comment by ronyeh** [comment]
Cool.

Can you explain what visual diffs the two metrics files changes introduce? I remember tweaking some of the metrics back in March to make stems align with note heads when zoomed in, and I'm wondering if / when we'll break those alignments again, haha. Maybe I need to add some tests to ensure the alignment doesn't break.

**Comment by AaronDavidNewman** [comment]
Is this comment for a different PR?  These are music theory tables in `Tables.ts` that I need to reference in Smoosic.  No changes at all, just exposing 3 functions through Flow namespace.

File: src/flow.ts [hunk 1]
```diff
@@ -231,4 +231,13 @@ export const Flow = {
   keySignature(spec: string): { type: string; line: number }[] {
     return Tables.keySignature(spec);
   },
+  clefProperties(clef: string): { line_shift: number } {
+    return Tables.clefProperties(clef);
+  },
+  keyProperties(key: string, clef?: string, params?: any): any {
```
Comment (rvilarl) [inline src/flow.ts:237]: Here eslint should be disabled. Place the following comment:
```
// eslint-disable-next-line
```

File: src/fonts/gonville_metrics.ts [hunk 2]
```diff
@@ -92,20 +92,20 @@ export const GonvilleMetrics = {
...
       spacing: 4,
-      offsetYStemUp: -9,
-      offsetYStemDown: -21,
-      offsetXStemUp: 6,
-      offsetXStemDown: -2,
+      offsetYStemUp: -17,
+      offsetYStemDown: -2,
+      offsetXStemUp: 6.5,
+      offsetXStemDown: -0.5,
     },
```
Comment (rvilarl) [inline src/fonts/gonville_metrics.ts:109]: This seems not to be part of this PR. Remove the changes.

File: src/fonts/petaluma_metrics.ts [hunk 3]
```diff
@@ -117,15 +117,15 @@ export const PetalumaMetrics = {
...
+      offsetXStemUp: 13,
       offsetXStemDown: 1,
     },
     grace: {
       point: 18,
       spacing: 4,
       offsetYStemUp: -5,
       offsetYStemDown: 5,
-      offsetXStemUp: 7,
+      offsetXStemUp: 8,
```
Comment (rvilarl) [inline src/fonts/petaluma_metrics.ts:128]: This file also seems to be included by mistake.

**Review by rvilarl — CHANGES_REQUESTED** [review]

**Comment by ronyeh** [comment]
> Is this comment for a different PR? These are music theory tables in `Tables.ts` that I need to reference in Smoosic. No changes at all, just exposing 3 functions through Flow namespace.

If you look at the files changed tab, it shows you submitted 3 files, including changes to the metrics. Rodrigo also pointed it out. We assume you just wanted to change flow.ts?

**Comment by AaronDavidNewman** [comment]
I did something weird, sorry. Let me try again.

**Comment by ronyeh** [comment]
You asked this question on the forums, but since you are submitting PRs from your master branch, you might have trouble submitting multiple PRs at the same time. Rodrigo and I each create new branches off of our own master for each feature or fix. Our master branch is the same as the 0xfe master branch.

**Comment by AaronDavidNewman** [comment]
right, I see that now.  I have to make a new branch each time, or at least it is easier that way.
